js/main.js

```diff
@@ -1,24 +1,25 @@
 function scrollToHash(hash) {
-  const offsetTop = -$('.top-bar').height();
```

Comment:
This offset should be kept, since otherwise the menu links are slightly off

Reply:
Done, although `.top-bar` is variable height on mobile. Changed to `.title-bar`.

Reply:
Done, although `.top-bar` is variable height on mobile. Changed to `.title-bar`.